docs/src/pages/guides/notifications.md

```diff
@@ -56,7 +56,7 @@ Notifications can be sent through multiple channels and mail accounts, each are
 
 ## Mailgun
 
-Mailgun is the notification channel through the mailgun email service. For this you need to have a mailgun account first, using your mailgun's API_KEY and DOMAIN you can send notifications to the recipient's mail.
+Mailgun is the notification channel through the mailgun email service. First, you need to have a mailgun account, then get the API key and DOMAIN. Please check out the [documentation in mailgun](https://documentation.mailgun.com/en/latest/quickstart.html). Put the API key and the DOMAIN in the monika's configuration as follows.
```

Comment:
Mailgun is an email notification delivery provided by Mailgun email service. To use mailgun for your notification, you 
 would need a mailgun account to obtain your API_KEY and DOMAIN. Please consult [mailgun documentation here](https://documentation.mailgun.com/en/latest/quickstart.html) on how to obtain them. Once that done, you could use the API_KEY and DOMAIN in monika's json configuration as follows;

Reply:
updated, please check